app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -1,4 +1,5 @@
 module DerivedVariables::LettingsLogVariables
+  # this is opaque enough to deserve a comment
```

Comment:
@kosiakkatrina it sounded when we discussed that you had an idea of why this is this way. I feel this could do with a comment so at least hopefully the next person to engage with this code knows that we all have sympathy with them and the sinking feeling in their stomach